ArduCopter/Copter.cpp

```diff
@@ -205,6 +205,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if STATS_ENABLED == ENABLED
     SCHED_TASK_CLASS(AP_Stats,             &copter.g2.stats,            update,           1, 100),
 #endif
+    SCHED_TASK(mcpUpdate, 10, 75),
```

Comment:
new sensors should be hooked into AP_Vehicle, not Copter, so they work on all vehicle types